openedx_tagging/core/tagging/rest_api/v1/views.py

```diff
@@ -107,6 +107,7 @@ class TaxonomyView(ModelViewSet):
 
     """
 
+    pagination_class = TaggingPagination
```

Comment:
Could you also please document the added `page` and `page_size` parameters for the List view?

Reply:
Done e45f79174b4a18a285f931c435b7f36251bb0243